SPECS/mysql/mysql.spec

```diff
@@ -98,6 +97,10 @@ fi
 %{_libdir}/pkgconfig/mysqlclient.pc
 
 %changelog
+* Fri Oct 18 2024 Sudipta Pandit <sudpandit@microsoft.com> - 8.0.37-1
+- Upgrade to 8.0.40 to fix CVE-2024-21096
```

Comment:
Can you list the other CVEs fixed by this upgrade? Or mention that it fixes 19 CVES

Reply:
It would be best to list all of the CVEs that we know are fixed